packages/main/src/Avatar.ts

```diff
@@ -276,6 +276,12 @@ class Avatar extends UI5Element implements ITabbable {
 
 	_onclick?: (e: MouseEvent) => void;
 	static i18nBundle: I18nBundle;
+	_handleResizeBound: () => void;
```

Comment:
There are a lot of components that needs to be refactored, but here as it's a new addition, we can reuse the `ResizeObserverCallback` type as type for the handler.
You can find examples in many components.

`import type { ResizeObserverCallback } from "@ui5/webcomponents-base/dist/delegate/ResizeHandler.js";`